src/Shared/PackageManagers/PyPIProjectManager.cs

```diff
@@ -496,17 +495,16 @@ public override Uri GetPackageAbsoluteUri(PackageURL purl)
                     }
                     else if (property.Name.Equals("project_urls"))
                     {
-                        foreach (var propertyName in searchPropertiesProjectURLs)
+                       foreach (JsonProperty project_url in property.Value.EnumerateObject())
                         {
-                            if (property.Value.TryGetProperty(propertyName, out JsonElement projectUrl))
+                            string projectUrl = project_url.Value.ToString() ?? string.Empty;
```

Comment:
Assuming we go this path, there's no need to call ExtractGitHubPackageURLs on string.empty, so this could be an if instead.

Reply:
I agree with you. Since I am not familiar with C#, I follow the code in Line 487. Shall I change both lines?